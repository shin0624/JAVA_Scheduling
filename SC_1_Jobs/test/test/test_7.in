1
10
A  B  C  D  E   F   G   H   I   J
0  4  7 15 15  23  24  24  37  38
3  6  4  5  2   6   3   4   5   2

0
