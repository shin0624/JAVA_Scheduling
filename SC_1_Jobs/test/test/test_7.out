************************ Main Menu *******************
* 0.Exit  1.Jobs 2.Process                           *
* 3.FCFS  4.SPN  5.HRRN  6.SRT  7.RR(q=1)  8.RR(q=4) *
******************************************************
Menu item number? 1
The number of processes? 10
input 10 process names: A B C D E F G H I J 
input 10 arrival times: 0 4 7 15 15 23 24 24 37 38 
input 10 service times: 3 6 4 5 2 6 3 4 5 2 

 A  B  C  D  E  F  G  H  I  J 
 0  4  7 15 15 23 24 24 37 38 
 3  6  4  5  2  6  3  4  5  2 

************************ Main Menu *******************
* 0.Exit  1.Jobs 2.Process                           *
* 3.FCFS  4.SPN  5.HRRN  6.SRT  7.RR(q=1)  8.RR(q=4) *
******************************************************
Menu item number? 0
Good bye.
